traits.yml

```diff
@@ -249,11 +549,236 @@ traits:
+        description: >
+          Any entity that holds audio data
+        usage:
+          - entity
+        traitSet:
+          - namespace: usage
+            name: Entity
+          - namespace: audio
+            name: Audio
+      SampledAudio:
+        description: >
+          Any entity that holds time-sampled audio data in a sidecar
+          resource.
+        usage:
+          - entity
+        traitSet:
+          - namespace: usage
+            name: Entity
+          - namespace: audio
+            name: Audio
+          - namespace: content
+            name: LocatableContent
+  function:
+    description: >
+      Specifications for entities that undertake specific functional roles.
+    members:
+      Workfile:
+        description: >
+          The entity is the product of some manual task or process,
+          that often defines how to produce or derive other data.
+
+
+          Common examples are the documents opened, worked on and saved
+          by artists using DCC tools to create models/images/etc.
+        usage:
+          - entity
+        traitSet:
+          - namespace: usage
+            name: Entity
+          - namespace: function
+            name: Work
+          - namespace: content
+            name: LocatableContent
+  threeDimensional:
+    description: >
+      Specifications for commonly encountered three-dimensional data types.
+    members:
+      Spatial:
+        description: >
+          Any entity that holds three-dimensional data.
+        usage:
+          - entity
+        traitSet:
+          - namespace: usage
+            name: Entity
+          - namespace: threeDimensional
+            name: Spatial
+      Scene:
+        description: >
+          Any entity that holds three-dimensional data in a sidecar resource.
+        usage:
+          - entity
+        traitSet:
+          - namespace: usage
+            name: Entity
+          - namespace: threeDimensional
+            name: Spatial
+          - namespace: content
+            name: LocatableContent
+      SceneGeometry:
+        description: >
+          Any entity that holds three-dimensional data describing one or
+          more objects, in a sidecar resource.
+        usage:
+          - entity
+        traitSet:
+          - namespace: usage
+            name: Entity
+          - namespace: threeDimensional
+            name: Spatial
+          - namespace: threeDimensional
+            name: Geometry
+          - namespace: content
+            name: LocatableContent
+      SceneCamera:
+        description: >
+          Any entity that holds a representation of a digital camera, in
+          a sidecar resource.
+        usage:
+          - entity
+        traitSet:
+          - namespace: usage
+            name: Entity
+          - namespace: threeDimensional
+            name: Spatial
+          - namespace: imaging
+            name: Camera
+          - namespace: content
+            name: LocatableContent
```

Comment:
[question] A bit too late for this, but it occurs to me that it's common in a node graph for a node to have a parameter referring to a scene path in the "upstream" scene.  So I'm wondering if hardcoding `LocatableContent` is the right thing to do, when you could swap it out with `ScenePath`?

Reply:
These would be two very different kinds of asset. These traits are specifically for where a tool needs to load such data from a resource to integrate into its scene. Added the [Resource]9https://github.com/OpenAssetIO/OpenAssetIO-MediaCreation/pull/71/commits/4fc14954a738b8bbd06023b11d835c282c541f1d) suffix to clarify.

Reply:
"Resource" suffix helps :+1: